xmlStyle.go

```diff
@@ -15,6 +15,44 @@ import (
 	"sync"
 )
 
+// Excel styles can reference number formats that are built-in, all of which
+// have an id less than 164. This is a possibly incomplete list comprised of as
+// many of them as I could find.
+var builtInNumFmt map[int]string = map[int]string{
+	0:  "general",
```

Comment:
the first `map[int]string` is redundant and can be skipped.

[example](http://play.golang.org/p/BzV7aRSqzb)